### PR TITLE
feat: rabble bounty

### PR DESCRIPTION
**Pull request by HAPPYS1NGH**
Created a new bounty for Telegram Mini Apps

### CODE REVIEW

File: telegram-mini-app-game/README.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+- 🧪 Robust testing for various edge cases
+- ✅ Include unit tests, integration tests, and end-to-end tests
+
+## 💰 Bounty Details
+
+Participants can earn up to **$ ** based on the implemented features.
+
+## 📤 Submission Guidelines
+
+1. Create an issue in the [bounty repository](https://github.com/spheronfdn/spheron-bounties) including:
```
Comment (izrake) [inline telegram-mini-app-game/README.md:73]: Hey @HAPPYS1NGH the issue submission link attached is wrong. It must be https://github.com/spheronFdn/sos-ai-bounty

Please correct it

Comment (HAPPYS1NGH) [reply telegram-mini-app-game/README.md]: Fixed that

**Review by izrake — APPROVED** [review]
